libs/platform/src/lib/components/form/radio-group/radio-group.component.html

```diff
@@ -40,13 +40,13 @@
         <div fd-form-item>
             <fdp-radio-button
                 [name]="name"
-                [value]="item.value ? item.value : item"
+                [value]="getLookupValue(item)"
```

Comment:
We need to solve this differently than calling methods from template

Reply:
here we are looping through list of items. this item will be available in template only. in what other way this item can be passed to method "lookupValue(item: any)" without calling method.
please suggest the way.